ts/src/bitmex.ts

```diff
@@ -977,6 +977,7 @@ export default class bitmex extends Exchange {
             'datetime': undefined,
             'nonce': undefined,
         };
+        let highestTimestamp = 0;
```

Comment:
for orderbook, we have timestamp for each ladder entry, so we have to locate the most recent timestamp.